x/tokenomics/keeper/settle_pending_claims.go

```diff
@@ -46,6 +46,9 @@ func (k Keeper) SettlePendingClaims(ctx sdk.Context) (numClaimsSettled, numClaim
 	logger.Info(fmt.Sprintf("found %d expiring claims at block height %d", len(expiringClaims), blockHeight))
 
 	for _, claim := range expiringClaims {
+		// TODO_DISCUSS_IN_THIS_PR: does the claim variable need to be copied here?
```

Comment:
TL;DR, I've added a test case to `settle_pending_claims_test.go` that covers the scenario where two claims are in the store and only one of them is expiring. I believe this case proves the rationale below.

As I'm looking at this again, I think there's no issue so long as we're not trying to pass the claim pointer to any async or deferred functions. I think the reason why I was originally concerned about this is because the existing telemetry calls I was referencing were all called deferred but since I opted out of that here, I don't think it's an issue after all. I think the same goes for all other receivers of a pointer to this claim value.